publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -52,25 +58,48 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
     @Value("${activedirectory.userSearchFilter}")
     private String userSearchFilter;
-    
+
     @Value("${activedirectory.rootDn}")
     private String rootDn;
 
-    @Override
-    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+    @Value("${activedirectory.connectionTimeOut}")
+    private String ldapTimeOut;
+
+//  built in connection timeout value for ldap if the network issue happens
+    public static final String DEFAULT_LDAP_CONNECTION_TIMEOUT = "12700";
```

Comment:
Why not defined as Integer as the other TCP timeout? And no quotes.

public static final Integer DEFAULT_TCP_TIMEOUT=60000;